packages/react-core/src/helpers/Popper/Popper.tsx

```diff
@@ -66,7 +81,7 @@ export interface PopperProps {
   /** popper direction */
   direction?: 'up' | 'down';
   /** popper position */
```

Comment:
I don't think left and right really are/should be deprecated in this case. I think there very well could be situations where left or right are what's actually desired rather than start and end.